pkg/virt-handler/libvirt/cache/cache.go

```diff
@@ -169,7 +169,7 @@ func NewDomain(dom kubevirt.VirDomain) (*kubevirt.Domain, error) {
 	}
 	return &kubevirt.Domain{
 		Spec: kubevirt.DomainSpec{},
-		ObjectMeta: kubeapi.ObjectMeta{
+		ObjectMeta: kubev1.ObjectMeta{
```

Comment:
This is the kubernetes one. I always use kube... for kuberntes packages and core... for ours (but the "core" term is outdated).